CDS/scripts/syndication_email.rb

```diff
@@ -165,20 +165,29 @@
 
 el = VSMetadataElements.new(vidispine_server,vidispine_port,username,passwd)
 
-wg = 'n/a'
-
-el.findUUID($store.substitute_string(ENV['working_group'])) do |entry|
-  puts entry.name
-  puts entry.uuid
-  entry.each do |item,values|
-    puts item
-    puts values
-    wg = values
-  end
+wg = $store.substitute_string(ENV['working_group'])
+
+colour_value = "af1674"
+colour_switch = rand 1..5
+
+if colour_switch == 1
```

Comment:
would be better written:

```ruby
case rand 1..5
  when 1
    colour_value="d10a10"
  when 2
    colour_value="ea5a0b"
...
end
```

It's more readable that way

see https://stackoverflow.com/questions/948135/how-to-write-a-switch-statement-in-ruby

Reply:
Done.